### DISCUSSION TITLE
Unable to stream

### DISCUSSION THREAD

**Discussion by sampsoe** [ant-media/Ant-Media-Server#5962] [Q&A]
Hello could any help or advise me on this please, every time time I click on "Start Broadcast" I get tge error, "Stream Start Failed, Error:null"  I am trying to stream to YouTube, all my settings seems to be ok, can anyone advised me please

Thank you.

**Comment by yashtandon113** [comment]
@sampsoe 
Please make sure that the URL format is correct in the playlist. Also, check out the below document for a playlist.
https://antmedia.io/docs/guides/publish-live-stream/playlist/

If all seems fine and you are still having an issue, please let us know.

**Comment by sampsoe** [comment]
Hello, am grateful for your advise,  I have checked my Playlist URL and it seems to be ok, this is the format of my play list url:  http://139.162.247.123:5080/LiveApp/Streams/610391713851488757377019.mp4      still have the same error.  My community version is: Community Edition 2.5.2 20230105_1947.


kind regards

**Reply by Mohit-3196** [reply]
Hi @sampsoe,
I just checked the URL and it seems the issue is that you are using Streams and not streams in the URL.

The correct path will be
http://139.162.247.123:5080/LiveApp/streams/610391713851488757377019.mp4

Let me know if it doesn't solve the issue.

Thank you,
Mohit

**Reply by MrCamotes** [reply]
I am having the same issue, but my path appears to be correct.  Not sure what I am doing wrong.

http://173.255.238.127:5080/LiveApp/streams/806834357509263976148843.mp4

**Reply by yashtandon113** [reply]
@MrCamotes 
When I tried to play your VOD URL, it was saying the resource is not available, which means this VOD/MP4 file is not available in LiveApp.

```
Type Status Report

Message The requested resource [/LiveApp/streams/806834357509263976148843.mp4] is not available

Description The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.
```

**Reply by MrCamotes** [reply]
I deleted the file and tried to reupload it, but now I am getting an error message after waiting for like 5 or 10 minutes for it to upload

**Reply by MrCamotes** [reply]
fyi - it is a regular mp4 file/video that i can click on and watch on my laptop - but for some reason it isn't uploading correctly

**Comment by sampsoe** [comment]
Oh, wow! thank you very much I got it sorted, really appreciate that very much, am happy now, thanks, have a nice day.

**Reply by Mohit-3196** [reply]
Hi @sampsoe,
You are most welcome :)
I'm happy that it's working now.

Have a great weekend.

**Comment by sampsoe** [comment]
Does anyone has a any suggestion for me please, I am doing a streaming on YouTube using the community edition, but am getting this error from YouTube "The stream's current bitrate (12522.86 Kbps) is higher than the recommended bitrate. We recommend that you use a stream bitrate of 6800 Kbps"  I had an extensive look in my streaming settings, but did not find anything about this error.  Your help will be greatly appreciated.

Thanks in advance.

**Reply by yashtandon113** [reply]
@sampsoe 
Please check my answer in the below discussion comment.
https://github.com/orgs/ant-media/discussions/5952#discussioncomment-7917875

I have shared the link there for the YouTube streaming uploading settings guidelines.

I hope it helps.

**Comment by sampsoe** [comment]
I have closed my account, thank you.
________________________________

**Comment by OscarTheWhiteHatHacker** [comment]
It is the same issue as @sampsoe 's original issue, the format of my link is http://151.236.219.253:5080/LiveApp/streams/BO15jZ7K2grV1711914526028.mp4
